packages/sdk/src/ApiLimitsHandler.ts

```diff
@@ -0,0 +1,35 @@
+import { RequestController } from './RequestController'
+
+export class ApiLimitsHandler {
+	constructor(private readonly requestController: RequestController) {}
+
+	private getLastResponseHeadersFromController(): Record<string, string> {
+		return this.requestController.getLastResponseHeaders()
+	}
+
+	private isLastResponseHeadersSet(): boolean {
+		return this.requestController.isLastReponseHeadersSet()
+	}
+
+	public getRateLimit(): number | boolean {
+		if (this.isLastResponseHeadersSet()) {
+			const rateLimit = this.getLastResponseHeadersFromController()['x-rate-limit-limit'] ?? false
+			if (rateLimit) {
+				return parseInt(rateLimit, 10)
+			}
+		}
+
+		return false
+	}
+
+	public getRateLimitRemaining(): number | boolean {
```

Comment:
Please make it return only number.
Drop returning boolean value, if `areLimitsAvailable` call is goign to return false then just return 0 - add in docs info that call to `areLimitsAvailable` should be always made before calling this method